### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by meta-rahul-kumar2**
Assignment 2

### CODE REVIEW

File: Fundamental/.project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (meta-manish) [inline Fundamental/.project]: We don't need to commit this file

File: Fundamental/.classpath [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (meta-manish) [inline Fundamental/.classpath]: No Need to add classpath file in source code

File: Fundamental/.settings/org.eclipse.jdt.core.prefs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+eclipse.preferences.version=1
```
Comment (meta-manish) [inline Fundamental/.settings/org.eclipse.jdt.core.prefs]: No need to add .setting files and folder in project

File: Fundamental/src/com/fcfs/service/FCFS.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+import java.util.Scanner;
+/**
+ * 
+ * @author Rahul
+ * Created 15 January 2020
+ * Provide complete functionality of FCFS scheduler
+ */
+
+public class FCFS {
+	public static void main(String srgs[]){
```
Comment (meta-manish) [inline Fundamental/src/com/fcfs/service/FCFS.java]: main Method should be call from class Named Main only

File: Fundamental/src/com/fcfs/service/FCFS.java [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+		FcfsMethods scheduler = new FcfsMethods(); 
+		Scanner sc = new Scanner(System.in);
+		int numProcess=0 ,processTimes=2 ,option ,flag=0;
```
Comment (meta-manish) [inline Fundamental/src/com/fcfs/service/FCFS.java]: Add proper space before and after =

File: Fundamental/src/com/fcfs/service/FCFS.java [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+		do{
+			try {
+				System.out.println("Enter the number of process:");
```
Comment (meta-manish) [inline Fundamental/src/com/fcfs/service/FCFS.java]: Create Constant class for all String messages

File: Fundamental/src/com/fcfs/service/FCFS.java [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+				numProcess = sc.nextInt();
+				flag=1;
+			} catch (Exception e) {
+				System.out.println(FcfsMessage.invalidProcessNum);
+			}
+			sc.nextLine();
+		}while(flag==0);
```
Comment (meta-manish) [inline Fundamental/src/com/fcfs/service/FCFS.java]: Use proper naming Conversion. flag is not proper name, Name of variable should be  more appropriate

File: Fundamental/src/com/fcfs/service/FCFS.java [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+/**
+ * 
+ * @author Rahul
+ * Created 15 January 2020
+ * Provide complete functionality of FCFS scheduler
+ */
+
+public class FCFS {
+	public static void main(String srgs[]){
+		FcfsMethods scheduler = new FcfsMethods();
```
Comment (meta-manish) [inline Fundamental/src/com/fcfs/service/FCFS.java]: This method is too big. Try to split this method .

File: Fundamental/src/com/numbers/service/MathOperation.java [hunk 9]
```diff
@@ -0,0 +1,51 @@
+package com.numbers.service;
+
+import java.util.Scanner;
+
+public class MathOperation {
+	public static void main(String args[]){
+		NumberCalc numCalc = new NumberCalc();
```
Comment (meta-manish) [inline Fundamental/src/com/numbers/service/MathOperation.java]: This method is too big

Comment (meta-rahul-kumar2) [reply Fundamental/src/com/numbers/service/MathOperation.java]: I have handle proper exception and created a menu driven program thats why this method is little bit long

**Comment by meta-manish** [comment]
Need to create new PR

**Pull request merged**